src/hints.rs

```diff
@@ -138,6 +141,7 @@ impl<'a> From<&'a NotificationHint> for MessageItem {
             &NotificationHint::Y(value)                 => (Y              .to_owned(), MessageItem::Int32(value)),
             &NotificationHint::Urgency(value)           => (URGENCY        .to_owned(), MessageItem::Byte(value as u8)),
             &NotificationHint::Custom(ref key, ref val) => (key            .to_owned(), MessageItem::Str(val.to_owned ())),
+            &NotificationHint::CustomInt(ref key, val)  => (key            .to_owned(), MessageItem::Int32(val)),
```

Comment:
I think it might be best to have `Box<ToMessageItem>` instead of using generics, because then we won't get into the trouble of having to specify a type for non-custom messages, see [here](https://play.rust-lang.org/?code=%23%5Bderive(Debug)%5D%0Apub%20enum%20Stuff%3CT%3E%20%7B%0A%20%20%20%20Simple%2C%0A%20%20%20%20Complex(T)%2C%0A%7D%0A%0Afn%20main()%20%7B%0A%20%20%20%20println!(%22%7B%3A%3F%7D%22%2C%20Stuff%3A%3ASimple)%3B%0A%7D&version=stable).